pkg/cmd/grafana-example-apiserver/deploy/linux/service.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: example-apiserver
+subsets:
+  - addresses:
+      - ip: 172.17.0.1 # this is the gateway IP in the "bridge" docker network
```

Comment:
would it also be useful to have an example in the base folder to deploy to a hosted cluster with a deployment similar to the [metrics-server](https://github.com/kubernetes-sigs/metrics-server/blob/master/manifests/base/deployment.yaml)?

Reply:
Running it in the cluster is totally possible, just somewhat slows down the development workflow. For one thing, one can't breakpoint inside a container running in the local cluster as easily.

I am remembering the tilt setup you had in erstwhile Grafana-apiserver. It worked great otherwise. Would you like that brought over here?